frontend/simple/views/containers/contributions/PaymentMethods.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div>
+    <i18n tag="legend" class="label">Payment method</i18n>
+    <ul class="field has-addons c-options" :aria-label="L('Payment Methods')">
+      <li class="control" v-for="method in methodsAvailable">
+        <button v-if="method.available"
+          class="button is-outlined"
+          :class="{ 'is-primary': active === method.name }"
+          :aria-pressed="active === method.name"
+          @click="$emit('change', method.name)"
```

Comment:
At the moment, clicking on a payment method closes the modal immediately, it shouldn't do that, it should only close the modal when "Save Pledge/Income Details" is clicked

Reply:
Are you sure? I can't replicate that bug here 😕

Reply:
One thing I noticed is "manual" is not the default one, I fixed it. But we can't close a modal without clicking the Save btn